modules/cnf-topology-aware-lifecycle-operator-create-policies.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-create-policies_{context}"]
+= Creating update policies for managed clusters
+
+The `PolicyGenTemplate` configuration manifest creates an {rh-rhacm} policy that updates the targeted clusters from the `ClusterVersion` manifest to the desired version, which then triggers a platform update.
```

Comment:
This is not at all accurate.
@serngawy / @imiller0 / @Missxiaoguo, could you kindly help with a proper phrasing for PGT?
Also, @amolnar-rh, do we need a PGT section here? Doesn't PGT have its own documentation?